src/examples/Ext.lean

```diff
@@ -1,19 +1,66 @@
 import challenge_notations
 import challenge_prerequisites
+import for_mathlib.universal_delta_functor.Ext
 
 /-!
 
-This file shows that `Ext` is a universal δ-functor (TODO)
+This file shows that `Ext` is a universal δ-functor
 and performs the computation `Ext^1(ℤ/nℤ, ℤ/nℤ) = ℤ/nℤ`.
 
 -/
 
 noncomputable theory
 
 open category_theory category_theory.limits opposite
+open bounded_homotopy_category bounded_derived_category
+
+section Ext
+
+universes v u
+-- Let's work with an abelian category which has enough projectives.
+variables {A : Type u} [category.{v} A] [abelian A] [enough_projectives A]
```

Comment:
I personally find it a tiny bit confusing if the category is called `A` and the second entry of `Ext(_, _)` is called `B`. My mind immediately thinks that `A` should be the first entry as well.
I guess the two obvious solutions are: use fancy curly letters for the category, or use `Y` for the snd entry.

Reply:
I used curly A's and changed the objects to X and/or Y.